### ISSUE TITLE
Dead link is not really dead

### ISSUE THREAD

**Issue by dialex** [remarkjs/remark-lint-no-dead-urls#12]
Your lib marks this link as dead https://vimeo.com/78912852 but it's not, as you can see by clicking it.

**Comment by wooorm** [comment]
@dialex Could you expand on this? [Here’s some more info on creating a good, actionable, issue](https://github.com/remarkjs/remark/blob/master/contributing.md#submitting-an-issue).

**Comment by dialex** [comment]
Nevermind, I created my own script to validate broken links.
This lib and [remark-validate-links](https://github.com/remarkjs/remark-validate-links) give me too much false positives (ie. marks valid links as dead). I'm sorry, this is too unreliable for me.

**Comment by NewAlexandria** [comment]
Can we re-open this issue?  Some vimeo links still are marked dead, when they are not. 

```
- Linting
✖ Linting
  README.md:212:3
  ✖  212:3  Link to https://vimeo.com/20241649 is dead  remark-lint:no-dead-urls
  ✖  213:3  Link to https://vimeo.com/9790850 is dead   remark-lint:no-dead-urls
  2 errors
```
@wooorm

**Comment by NewAlexandria** [comment]
@dialex I don't see your link validation lib.  Can you post / package it, please?

**Comment by davidtheclark** [comment]
See https://github.com/transitive-bullshit/check-links/issues/4. check-links is the lower-level link-checking library this plugin uses. I'm sure they'd appreciate a PR, if @dialex or anybody else has ways to fix some false positives.

**Comment by dialex** [comment]
@NewAlexandria  My bad, it was in a branch. It's merged now, [this is the file](https://github.com/dialex/start-testing/blob/master/scripts/link-validator.js). I asked a friend, it took him a couple of hours, and it works (so far no false positives) 👍 

To run it: `node scripts/link-validator.js docs/**/*.md` (docs is my folder with markdown files)

**Comment by fregante** [comment]
I'm having the same issue here: https://travis-ci.org/fregante/Awesome-WebExtensions/jobs/563973345#L208

It says https://bfred.it is not working (both on Travis and my computer) but the URL loads fine

**Comment by wooorm** [comment]
@fregante please post an issue on the underlying project, mentioned above.
